src/Uno.UI/UI/Xaml/Extensions/FontHelper.Lookup.Android.cs

```diff
@@ -55,19 +57,19 @@ public override bool Equals(object other)
 
 			private readonly HashtableEx _entries = new HashtableEx();
 
-			internal bool TryGetValue(Entry key, out Typeface result)
+			internal bool TryGetValue(Entry key, out Typeface? result)
 			{
 				if (_entries.TryGetValue(key, out var value))
 				{
-					result = (Typeface)value;
+					result = (Typeface?)value;
 					return true;
 				}
 
 				result = null;
 				return false;
 			}
 
-			internal void Add(Entry key, Typeface typeFace)
+			internal void Add(Entry key, Typeface? typeFace)
```

Comment:
I do not think add the null object to `_entries` is a good idea. It will make the TryGetValue can find from the `_entries` but return a null Typeface.

Reply:
That's the idea. Some typefaces cannot be found and will return a cached null value, there's no point in making the looking multiple times.